audit/projects/k8s-staging-cluster-api-gcp/iam.json

```diff
@@ -1,4 +1,34 @@
 {
+  "auditConfigs": [
+    {
+      "auditLogConfigs": [
+        {
+          "logType": "ADMIN_READ"
+        },
+        {
+          "logType": "DATA_READ"
+        },
+        {
+          "logType": "DATA_WRITE"
+        }
+      ],
+      "service": "iam.googleapis.com"
+    },
+    {
+      "auditLogConfigs": [
+        {
+          "logType": "ADMIN_READ"
+        },
+        {
+          "logType": "DATA_READ"
+        },
+        {
+          "logType": "DATA_WRITE"
+        }
+      ],
+      "service": "compute.googleapis.com"
+    }
+  ],
```

Comment:
This is me doing something manually: https://github.com/kubernetes/k8s.io/pull/2147#issuecomment-871854364